WebRoot/js/zimbraMail/prefs/view/ZmFilterRuleDialog.js

```diff
@@ -578,6 +595,7 @@ function(conf, field, options, rowData, testType, rowId) {
 			else if (testType == ZmFilterRule.TEST_CONVERSATIONS || testType == ZmFilterRule.TEST_LIST  ||  testType == ZmFilterRule.TEST_BULK || testType == ZmFilterRule.TEST_IMPORTANCE || testType == ZmFilterRule.TEST_FLAGGED) {
 				select.addChangeListener(this._importanceChangeLstnr);
 			}
+			select.setAriaLabel("value");
```

Comment:
"Value"
although it won't be visible let's stick to using titlecase

Reply:
Updated